src/components/text-input.components.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable default-param-last */
+/* eslint-disable unicorn/better-regex */
+import $ from 'jquery';
+
+export default class TextInput {
+    constructor(stringQuery, textEnable = false, required = true, textValid = '') {
+        this.stringQuery = stringQuery;
+
+        this.textInput = $(`${stringQuery} input[type="text"]`);
+
+        this.textValid = textValid;
+        this.textEnable = textEnable;
+        this.required = required;
+    }
+
+    getTextInput() {
+        return this.textInput.val();
+    }
+
+    setTextInput(value) {
+        this.textInput = this.textInput.val(value);
+    }
+
+    getTextValid() {
+        return this.textValid.val();
+    }
+
+    setTextValid(text) {
+        this.textValid = this.textInput.val(text);
+    }
+
+    isTextValid() {
+        if (this.textInput.val() === '') {
+            throw new Error('write any thing');
+        }
+
+        return this.textInput.val();
+    }
+
+    isTextEnable() {
+        if (!this.textInput.prop('disabled')) {
+            return true;
+        }
+
+        return false;
+    }
+
+    getTextRequired() {
+        return this.textInput.prop('required');
+    }
+
+    setTextRequired(boolean) {
+        this.textInput.prop('required', boolean);
+        this.required = boolean;
+    }
+
+    isTextRequired() {
+        if (!this.textInput.prop('required')) {
+            this.required = false;
+            return this.required;
+        }
+
+        return this.required;
+    }
+}
```

Comment:
Add a blank line at the end of the file

Reply:
![image](https://user-images.githubusercontent.com/62728106/163028880-f0d5900f-55c6-4247-8fe2-66ecccad16bc.png)
fixed